towhee/functional/mixins/milvus_mixin.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from towhee.hparam import param_scope
+from ..entity import Entity
+
+
+def insert(self, index, collection, batch_size):
+    # pylint: disable=import-outside-toplevel
+    from towhee.utils.milvus_utils import Collection, MutationResult
+    if isinstance(collection, str):
+        collection = Collection(collection)
+    primary_keys = []
+    insert_count = 0
+    data = [[] for _ in range(len(index))]
+
+    for entity in self:
+        for i in range(len(index)):
+            data[i].append(getattr(entity, index[i]))
+        if len(data[0]) == batch_size:
+            mr = collection.insert(data)
+            data = [[] for _ in range(len(index))]
+            primary_keys += mr.primary_keys
+            insert_count += mr.insert_count
+            print(f'Successfully inserted {mr.insert_count} row data.')
```

Comment:
Log messages should be printed into stderr. And the message should be controlled by some options.

Please consider using engine log for the message

Reply:
Updated with `engine_log.info`, but it will not print without if not set `engine_log.setLevel('INFO')`. Another PR will submit to add `towhee.set_log_levle` for configuration.